com.lakoliu.Furtherance.json

```diff
@@ -0,0 +1,114 @@
+{
+    "app-id" : "com.lakoliu.Furtherance",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "furtherance",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.gnome.Mutter.IdleMonitor"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "furtherance=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name":"libsass",
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://github.com/sass/libsass/archive/3.6.4.tar.gz",
+                    "sha256":"f9484d9a6df60576e791566eab2f757a97fd414fce01dd41fc0a693ea5db2889"
+                },
+                {
+                    "type":"script",
+                    "dest-filename":"autogen.sh",
+                    "commands":[
+                        "autoreconf -si"
+                    ]
+                }
+            ]
+        },
+        {
+            "name":"sassc",
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://github.com/sass/sassc/archive/3.6.1.tar.gz",
+                    "sha256":"8cee391c49a102b4464f86fc40c4ceac3a2ada52a89c4c933d8348e3e4542a60"
+                },
+                {
+                    "type":"script",
+                    "dest-filename":"autogen.sh",
+                    "commands":[
+                        "autoreconf -si"
+                    ]
+                }
+            ]
+        },
+        {
+            "name":"gtk",
+            "buildsystem":"meson",
+            "config-opts":[
+                "-Dbuild-examples=false",
+                "-Dbuild-tests=false",
+                "-Dintrospection=disabled"
+            ],
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz",
+                    "sha256":"782d5951fbfd585fc9ec76c09d07e28e6014c72db001fb567fff217fb96e4d8c"
+                }
+            ]
+        },
+        {
+            "name":"libadwaita",
```

Comment:
libadwaita-1 1.1 is in the runtime

Reply:
an REMOVE THIS, and the dependencies

Reply:
Okay, sorry about that, I've removed both.